lib/con_man.rb

```diff
@@ -84,7 +84,7 @@ def valid_email?
   end
 
   def click
-    Contacts.create(:first_name => find_widget("first_name").text)
+    Contacts.create(:first_name => find_widget("first_name").text, :last_name => find_widget("last_name").text, :dob => find_widget("dob").text, :telephone => find_widget("telephone").text, :email=> find_widget("email").text)
```

Comment:
Long line
You should create a newline after each comma, and inline to one tabstop (don't be tempted to line up the colons)